script/proto-docs.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -eux -o pipefail
+
+mkdir -p docs/grpc
+
+DOCROOT="${DOCROOT:$PWD}"
+target=docs/grpc
+templates=docs/grpc-templates
+filemap="$templates/filemap.xml"
+
+cp "$templates/index-header.html" "$target/index.html"
+
+
+for pkg in \
+    'api/events' \
+    'api/types' \
+    'api/types/task' \
+    'api/services/images/v1/' \
+    'api/services/tasks/v1/' \
+    'api/services/content/v1/' \
+    'api/services/snapshot/v1/' \
+    'api/services/namespaces/v1/' \
+    'api/services/diff/v1/' \
+    'api/services/version/v1/' \
+    'api/services/events/v1/' \
+    'api/services/leases/v1/' \
+    'api/services/introspection/v1/' \
+    'api/services/containers/v1/' \
+; do
+    protoc \
```

Comment:
This is really a job for protobuild. We may need to make changes to the tool but I really don’t want to start using any scripts for this kind of thing when we already have a tool in place.